status/weather.conf

```diff
@@ -3,7 +3,7 @@
 
 # Requires https://github.com/xamut/tmux-weather.
 
-set -ogq @catppuccin_weather_icon ""
+set -ogq @catppuccin_weather_icon " "
```

Comment:
nit: These icons aren't showing for me in chrome when reviewing changes. Maybe it's worth changing the icons to use hex codes (not in this change)

Reply:
I would prefer to keep it as icon's so that people can directly copy/see it, hex code adds another barrier.

Reply:
I would prefer to keep it as icon's so that people can directly copy/see it, hex code adds another barrier.